docs/API/enable-sitemap-scheduler.code.js

```diff
@@ -0,0 +1,12 @@
+const response = await client.enableSitemapScheduler(123, {
+	"cron_minute": "*/10",
```

Comment:
I think object keys should be without double quotes, for example, Create Scraping Job section it's similar request and example is without double quotes.

Reply:
Yes, You are right, missed that